Docs: About Caesar Cipher

jaechanjo opened the pull request:
Refered by Wikipedia, summarize definition and
example of Caesar Cipher.

jaechanjo committed:
Docs: About Caesar Cipher

Refered by Wikipedia, summarize definition and
example of Caesar Cipher.

dahee-diana96 commented on Caesarcipher.md:
```diff
@@ -0,0 +1,3 @@
+## Caesar cipher
+
+암호법의 일종으로 카이사르 암호, 변환 암호, 카이사르 변환으로 알려져 있다. 가장 단순하면서 가장 널리 사용되는 암호기법이다. 특정 위치의 알 파벳이 같은 방식으로 몇 칸 밀려서 대응되는 방식이다. 예를 들자면, 3 칸 왼쪽으로 변환한다면, D는A로, E는 B로 대응되는 식이다.
```
설명 항목과 예시 항목으로 분리해보는 건 어떨까용?

jaechanjo replied:
아하 좋은 말씀이십니당!

dahee-diana96 replied:
한결 나아보이는군용! :)

jaechanjo committed:
Refactor: Composing good stucture of paragraph.

Because of bad structure of paragraph,
mixing explanation with example,
It was composed separately for read easily

dahee-diana96 reviewed (approved):
That looks better!
